src/detect-tcpmss.h

```diff
@@ -26,4 +26,6 @@
 
 void DetectTcpmssRegister(void);
 
+const char *DetectTcpmssModeToString(uint8_t mode);
```

Comment:
nit: function signatures are not matching :P

Reply:
Sheesh!

🤣.

I'll sort that.